src/v/storage/disk_log_impl.cc

```diff
@@ -523,12 +523,19 @@ segment_set disk_log_impl::find_sliding_range(
 
 ss::future<bool> disk_log_impl::sliding_window_compact(
   const compaction_config& cfg, std::optional<model::offset> new_start_offset) {
-    vlog(gclog.info, "[{}] running sliding window compaction", config().ntp());
+    vlog(gclog.debug, "[{}] running sliding window compaction", config().ntp());
     auto segs = find_sliding_range(cfg, new_start_offset);
     if (segs.empty()) {
-        vlog(gclog.info, "[{}] no more segments to compact", config().ntp());
+        vlog(gclog.debug, "[{}] no more segments to compact", config().ntp());
         co_return false;
     }
+    vlog(
+      gclog.debug,
+      "[{}] compacting {} segments in interval [{}, {}]",
+      config().ntp(),
+      segs.size(),
+      segs.front()->filename(),
+      segs.back()->filename());
```

Comment:
Are all 3 of these required? Can `find_sliding_range` throw?

Reply:
>Are all 3 of these required

I think it's a useful identifier of the compaction. Arguably you can deduce this from the set of logs that will proceed, but as far as debugging is concerned, a summary/expectation is a nice to have

>Can find_sliding_range throw?

No, it just iterates through segments and adds them to a segment_set